docs: revised the drag and drop feature page (#25939)
electron/electron: docs/tutorial/native-file-drag-drop.md

# Native File Drag & Drop

## Overview

Certain kinds of applications that manipulate files might want to support
the operating system's native file drag & drop feature. Dragging files into
web content is common and supported by many websites. Electron additionally
supports dragging files and content out from web content into the operating
system's world.

> To implement this feature in your app, you need to call

Reviewer: ```suggestion
To implement this feature in your app, you need to call the
```

[`webContents.startDrag(item)`](../api/web-contents.md#contentsstartdragitem)
API in response to the `ondragstart` event.

## Example

> Assuming you have a working Electron application from the
> [Quick Start Guide](quick-start.md), add the following lines to the

Reviewer 2: Linking back to quick-start is a nice edition; someone landing on this page will be able to walk back to a starting point & be able to make running code.

I can imagine this preface becoming repetitive if we add it to more and more code samples, so I golfed it slightly:

```suggestion
Starting with a working application from the
[Quick Start Guide](quick-start.md), add the following lines to the
```

`index.html` file:

```html
<a href="#" id="drag">Drag me</a>
<script src="renderer.js"></script>
```

> and add the `renderer.js` file:

Reviewer 2: first-app.md points users to electron-quick-start which already has a placeholder `renderer.js`

```suggestion
and add the following lines to the `renderer.js` file:
```

  })
})
```

> After launching the Electron application, try to drag-and-drop the item.

Reviewer: ```suggestion
After launching the Electron application, try dragging and dropping the item from the app's
BrowserWindow onto your desktop.
```
